nemo_skills/evaluation/evaluator/math.py

```diff
@@ -58,30 +60,108 @@ class LeanEvaluatorConfig:
 
 
 def eval_lean4_proof(cfg):
-    eval_config = LeanEvaluatorConfig(**cfg.eval_config)
-
-    sandbox = get_sandbox(**eval_config.sandbox)
-    eval_config_dict = asdict(eval_config)
-    eval_config_dict.pop("sandbox")
-    asyncio.run(
-        sandbox.batch_evaluate_results(
-            input_files=cfg.input_files,
+    """Legacy function - delegates to Lean4ProofEvaluator class."""
```

Comment:
why do we need these two if they are already defined as classes?

Reply:
Just in case any users of NeMo-Skills imports them somewhere it doesn't break their code. But we can also remove, they're not needed internally afaik

Reply:
let's remove, these shouldn't be used anywhere